tests/closure/tconstclosure.nim

```diff
@@ -0,0 +1,47 @@
+
+discard """
+  cmd: "nim check --hints:off --warnings:off $file"
+  action: "reject"
+  nimout: '''
+tconstclosure.nim(44, 24) Error: const closure proc with an environment cannot be used at runtime: makeMyClosure(myRef)
+
+
+
+
+
+
```

Comment:
Why are there lots of blank lines?

Reply:
I see some test file like this, thought it be usefull when you want add more test cases in same file, and without modify the nimout file line number.